src/components/LoginModal/LoginModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {
+  LoginWrap,
+  LoginBox,
+  CloseBtn,
+  Content,
+  LoginBtn,
+} from './LoginModal.styles'
+
+interface ModalProps {
+  setModalOpen: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+const LoginModal = ({ setModalOpen }: ModalProps) => {

```

Comment:
(p5) 아래와 같이 쓰기도 하는데 이 부분은 팀의 컨벤션으로 통일되면 될 것 같네요
성능적 차이는 없습니다~
```suggestion
const LoginModal: React.FC<ModalProps> = ({ setModalOpen }) => {
```